model/Security/Classes/VexAffectedVulnAssessmentRelationship.md

```diff
@@ -4,21 +4,37 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+Connects a vulnerability and an element designating the element as a product
+affected by the vulnerability.
 
 ## Description
 
-An VexAffectedVulnerabilityCategorization is TODO
+`VexAffectedVulnAssessmentRelationship` connects a vulnerability and a number
+of elements. The relationship marks these elements as products affected by the
+vulnerability. This relationship corresponds to the VEX `affected` status.
 
-## Metadata
+## Constraints
 
-- name: VexAffectedVulnAssessmentRelationship
-- SubclassOf: /Security/VexVulnAssessmentRelationship 
-- Instantiability: Concrete
+When linking elements using a `VexAffectedVulnAssessmentRelationship`, the
+following requirements must be observed:
 
-## Properties
+* Elements linked with a `VulnVexAffectedAssessmentRelationship` are constrained
+to the `affects` relationship type.
 
-- actionStatement
-  - type: xsd:string
-  - minCount: 1
-  - maxCount: 1
+## Metadata
+* name: VexAffectedVulnAssessmentRelationship
```

Comment:
Since these files are parsed with https://github.com/spdx/spec-parser, which might not be stable or resilient regarding the types of list, please use `-` instead of `*` to stay very similar to the other files in the repo

Reply:
updated